frontend/.vscode/settings.json

```diff
@@ -59,6 +59,12 @@
   "files.associations": {
     "**/console-extensions.json": "jsonc"
   },
+  "json.schemas": [
+    {
+      "fileMatch": ["**/console-extensions.json"],
+      "url": "./packages/console-dynamic-plugin-sdk/dist/schema/console-extensions.json"
```

Comment:
Reference: https://code.visualstudio.com/docs/languages/json#_mapping-to-a-schema-in-the-workspace

The `./` prefix represents the workspace root.